<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectMakefileWidget</class>
 <widget class="QWidget" name="ProjectMakefileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpCustomMakefile">
     <property name="title">
      <string>Use custom make file</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="txtCustomMakefile"/>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="btnBrowse">
        <property name="toolTip">
         <string>Browse</string>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/images/newlook24/053-open.png</normaloff>:/icons/images/newlook24/053-open.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnInfo">
     <property name="text">
      <string>Information about using a custom make file </string>
     </property>
     <property name="icon">
      <iconset resource="../icons.qrc">
       <normaloff>:/icons/images/newlook24/085-tutor.png</normaloff>:/icons/images/newlook24/085-tutor.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Include the following files into the makefile:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
